python/chemiscope/explore.py

```diff
@@ -197,6 +198,121 @@ def soap_pca_featurize(frames, environments=None):
     return pca.fit_transform(feats)
 
 
+def metatensor_featurizer(model, extensions_directory="extensions/", device=None):
+    """
+    Create a featurizer function using a `metatensor`_ model to obtain the features
+    from structures with a pre-trained ``metatensor`` model.
+
+    Note: a model output is defined per-atom by default. The additional check of
+    consistency is run to verify the model.
```

Comment:
model should run per frame if the user wants frame features & per atom if there are environements